cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml

```diff
@@ -42,10 +42,10 @@ spec:
           value: --no-supervisor -q
         resources:
           limits:
-            memory: 300Mi
+            memory: 300Mi # fluentd-gcp-memory-limit
```

Comment:
If we decide to use this approach please add appropriate comments explaining the reason behind having this notation.